server/src/dto/organization.ts

```diff
@@ -0,0 +1,34 @@
+import { Transform } from 'class-transformer';
```

Comment:
Shall we rename this file to `organization.dto.ts`

Reply:
yeah ok, will fix